2015/proposals/banaszkiewicz-piotr-amy.md

```diff
@@ -0,0 +1,214 @@
+# Enhance Amy, a workshop-management platform for Software Carpentry
+
+## Abstract
+
+The number of Software Carpentry's workshops run weekly dynamically grows. All
```

Comment:
On Tue, Mar 24, 2015 at 01:32:00PM -0700, Piotr Banaszkiewicz wrote:
> ![index](https://cloud.githubusercontent.com/assets/72821/6812142/2adaf08e-d26d-11e4-8921-f1b536f1acff.png)

You need to sort your data to avoid the downward jiggle around April
2014.